.github/workflows/review.yml

```diff
@@ -0,0 +1,61 @@
+name: Build & Review
+on: [pull_request]
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+  CONTAINER_REGISTRY_USER: ${{ secrets.GHCR_CONTAINER_REGISTRY_USER }}
+  CONTAINER_REGISTRY_PASSWORD: ${{ secrets.GHCR_TOKEN }}
+  CONTAINER_REGISTRY_REPO: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+  REPO_NAME: ${{ github.event.repository.name }}
+  KUBECONFIG_RAW: ${{ secrets.KUBECONFIG_RAW }}
+  BUILD_ARTIFACT_FOLDER: "build_artifacts"
+  SERVICE_ARTIFACT_FOLDER: "service_artifacts"
+  BASE_DOMAIN: ${{ secrets.BASE_DOMAIN_STAGING }}
+  APP_MIGRATE_COMMAND: /app/.prod/on_deploy.sh
+  APP_INITIALIZE_COMMAND: /app/.prod/on_initialize.sh
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        uses: andersinno/kolga-build-action@v2
+
+  review:
+    runs-on: ubuntu-latest
+    needs: build
+    name: Review
+    steps:
+      - uses: actions/checkout@v2
+      - uses: andersinno/kolga-setup-action@v2
+
+      - name: Review-Services
+        uses: andersinno/kolga-deploy-service-action@v2
+        with:
+          projects: BERTH_RESERVATIONS
+        env:
+          POSTGRES_IMAGE: "docker.io/andersinnovations/postgis:11-bitnami"
```

Comment:
What's the default user it generates? We should probably also add the user/pass to the env variables on the `Deploy` action so it can connect to the db.

Reply:
You can use add DATABASE_USER & DATABASE_PASSWORD env variables if you want to use your own custom credentials. Otherwise it will auto-generate these and inject them to review deployment.

Reply:
Hmmm... there might be some misconfiguration with the variable names then. The review pipeline failed since it didn't get the right user/password.

https://github.com/City-of-Helsinki/berth-reservations/pull/399/checks?check_run_id=1536161086
```log
django.db.utils.OperationalError: FATAL:  password authentication failed for user "user"
```

Reply:
Oh, yeah actually we found bug there. When you first deploy the review env it works fine with the autogenerated user/password but as you make another push to your PR it triggers the pipeline again and redeploys the application but the updated credentials doesn't get reflected to postgresql pod and it still uses the old credentials. So the preferred way is to use those variables until we get this fixed on Kolga side.

Reply:
Sneaky bug! Haha but good find. So do we add manually the variables/credentials?

Reply:
Yes manually. I added them to the review workflow. They can be adjusted to your liking.